docs/concepts/operator.md

```diff
@@ -165,6 +168,72 @@ This range is used by the API server to compute the cluster IPs of `Service`s.
 
 The controller maintains the `.status.lastOperation` which indicates the status of an operation.
 
+##### Dashboard
+
+`.spec.virtualCluster.gardener.gardenerDashboard` serves a few configuration options for the dashboard.
+This section highlights the most prominent fields:
+
+1. `frontendConfigMapRef`: Reference a `ConfigMap` here that contains the frontend configuration, for example
+   ```yaml
+   apiVersion: v1
+   kind: ConfigMap
+   metadata:
+     name: gardener-dashboard-frontend
+     namespace: garden
+   data:
+     config.yaml: |
```

Comment:
See last commits - the data key was renamed to `frontend-config.yaml` and yes, sure, it must be like this. Otherwise GOP cannot understand it/does not know where to read the config from. This is already visible in the example - why/where to duplicate it?

Reply:
OK, done in last commit